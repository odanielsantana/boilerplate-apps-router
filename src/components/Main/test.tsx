import { render, screen } from '@testing-library/react'
import Main from '.'

describe('<Main />', () => {
  it('shoud render the heading', () => {
    // Renderiza o componente
    // Busca o elemento e verifica a existência

    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /react avançado/i })
    ).toBeInTheDocument()

    // Gerar Snapshots
    expect(container.firstChild).toMatchSnapshot()
  })
})
