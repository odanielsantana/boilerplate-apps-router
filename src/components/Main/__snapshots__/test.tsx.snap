// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> shoud render the heading 1`] = `
<main>
  <h1>
    React Avan√ßado
  </h1>
</main>
`;
